cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -198,7 +198,7 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           -- version that this node will declare that it understands, when it
           -- is in the Alonzo era. That is, it is the version of protocol
           -- /after/ Alonzo, i.e. Babbage.
-          alonzoProtVer = ProtVer 6 0,
+          alonzoProtVer = ProtVer 7 0,
```

Comment:
why is alonzo era version changed here?? This breaks existing explorers on public testnet that do check on versions...

Reply:
Yes, this indeed is creating a mess checking block version for specific era

Reply:
This change looks correct to me. We want we the nodes in the Alonzo era that are prepared for the upgrade to signal that they are ready by putting major protocol number 7 in the block header (since 7 corresponds to the next ledger era, Babbage).

Reply:
The change looks correct for future (babbage era), but it seems to be at the expense of current (future past) - using incorrect / vague version in block header that is kinda unrelated to actual protocol major version.

Thus, the entire mapping of protocol version that so far has been decisive between forks is now different based on the node version (not protocol_major as the block header suggests)?

Reply:
This is actually how it was designed to work from the very beginning, going back to the Shelley ledger spec. See section 13 of https://hydra.iohk.io/job/Cardano/cardano-ledger/shelleyLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec.

The version listed in the block header indicates the latest version of the protocol that the block producer is prepared to upgrade to.

Reply:
Thanks for the link, looks like I have been understanding the value of this block header wrong the whole time 😢  Looks like anyone with similar understanding might need immediate fixes. Can we add a reference to this (if not a short description) to the CIP 274?

Reply:
>  Can we add a reference to this (if not a short description) to the CIP 274?

That is a most excellent idea!